docs/operation/volume_management/volume_creation_deletion_gui.rst

```diff
@@ -0,0 +1,56 @@
+Volume Management using the UI
+==============================
+
+This section describes the creation and deletion of MetalK8s **Volume**
+using the MetalK8s UI.
+In order to create Volumes you need to have StorageClass objects registered in
+your cluster. See :doc:`/operation/volume_management/storageclass_creation`
+
+Volume Creation
+---------------
+
+To access the UI, refer to :ref:`this procedure <quickstart-services-admin-ui>`
+
+#. Navigate to the **Nodes** list page, by clicking the button in the sidebar:
+
+    .. image:: /operation/volume_management/img/node_list.png
+
+#. From the Node list, select the node you would like to create
```

Comment:
I guess we need a `period` or a `colon` at the end.

Reply:
Looks like we are not allowed to use period in a list.

I have had to check this too after some review from @slaperche-scality 

FYI: https://data.grammarbook.com/blog/colons/colons/